docs/guides/pgbouncer/monitoring/grafana-dashboard.md

```
@@ -0,0 +1,363 @@
+title: Visualize PgBouncer Metrics with Grafana Dashboard
+menu:
+  docs_{{ .version }}:
+    identifier: pb-grafana-dashboard-monitoring
+    name: Grafana Dashboard
+    parent: pb-monitoring-pgbouncer
+    weight: 25
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)